### PR TITLE
Flexbox/karma/bani

### PR DESCRIPTION
**Pull request by banirezaie**
a try to rebuild a simple website

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -16,4 +18,147 @@ body {
...
+    margin-top: 17.5rem;
+    font-size: 3.125rem;
+}
+.advertise{
+    /* padding: 10; */
+    /* margin-top: -800%; */
+}
+.buttons{
+    color: white;
+    /* display:flex;
```
Comment (ChrisOwen101) [inline css/style.css]: It's normally a good idea to remove comments when you're sure they're not needed, it stops the code from getting messy!

File: index.html [hunk 2]
```diff
@@ -17,5 +17,61 @@
...
...TElDICItLy9XM0MvL0RURCBTVkcgMS4xLy9FTiIgImh0dHA6Ly93d3cudzMub3JnL0dyYXBoaWNzL1NWRy8xLjEvRFREL3N2ZzExLmR0ZCI+DQo8c3ZnIHZlcnNpb249IjEuMSIgaWQ9IkxheWVyXzEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHg9IjBweCIgeT0iMHB4Ig0KCSB2aWV3Qm94PSIwIDAgMTQ0IDIxNiIgZW5hYmxlLWJhY2tncm91bmQ9Im5ldyAwIDAgMTQ0IDIxNiIgeG1sOnNwYWNlPSJwcmVzZXJ2ZSI+DQo8Zz4NCgk8cGF0aCBmaWxsPSIjRjE1QjJBIiBkPSJNMTI2LjEsMjEyLjljNC4xLDQuMSwxMC43LDQuMSwxNC44LDBjNC4xLTQuMSw0LjEtMTAuNywwLTE0LjhsLTI4LjYtMjguNmwxOC4xLTE4LjENCgkJYzguNC04LjQsMTMuNi0yMCwxMy42LTMyLjlDMTQ0LDkyLjgsMTIzLjIsNzIsOTcuNSw3MkM3MS44LDcyLDUxLDkyLjgsNTEsMTE4LjVjMCwxMi44LDUuMiwyNC41LDEzLjYsMzIuOWgwbDE4LjEsMTguMWwtMTguMSwxOC4xDQoJCWMtNC42LDQuNi0xMSw3LjUtMTguMSw3LjVjLTE0LjEsMC0yNS42LTExLjUtMjUuNi0yNS42di0xNTlDMjAuOSw0LjcsMTYuMiwwLDEwLjUsMEM0LjcsMCwwLDQuNywwLDEwLjV2MTU5DQoJCUMwLDE5NS4yLDIwLjgsMjE2LDQ2LjUsMjE2YzEyLjgsMCwyNC41LTUuMiwzMi45LTEzLjZsMTguMS0xOC4xTDEyNi4xLDIxMi45eiBNNzkuNCwxMzYuNmMtNC42LTQuNi03LjUtMTEtNy41LTE4LjENCgkJYzAtMTQuMSwxMS41LTI1LjYsMjUuNi0yNS42YzE0LjEsMCwyNS42LDExLjUsMjUuNiwyNS42YzAsNy4xLTIuOSwxMy41LTcuNSwxOC4xaDBsLTE4LjEsMTguMUw3OS40LDEzNi42eiIvPg0KPC9nPg0KPC9zdmc+DQo=" alt="logo"/></div>
```
Comment (ChrisOwen101) [inline index.html]: Instead of adding an image like this it's best to use a relative url e.g.

```
"../img/logo.svg"
```

File: index.html [hunk 3]
```diff
@@ -17,5 +17,61 @@
...
+            <figcaption>
+                Pay as you go
+            </figcaption>
+        
+        </figure>    
+            <p>
+            </p>
+            <section class="services">
+                <p>
```
Comment (ChrisOwen101) [inline index.html]: If these <p> tag aren't needed then they should be removed to make this a little cleaner!

File: index.html [hunk 4]
```diff
@@ -17,5 +17,61 @@
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
+    <nav class="navbar">
```
Comment (ChrisOwen101) [inline index.html]: Some of the code in here is a little messy and could be indented better. Fortunately Visual Studio Code can do this for you.

You can find out here here:
https://stackoverflow.com/a/29973358

File: css/style.css [hunk 5]
```diff
@@ -16,4 +18,147 @@ body {
...
+.logo{
+    height: 4.375rem;
+    width: 26px;
+}
+.navigates:hover{
+    cursor: pointer;
+    color: #333333;
+}
+.section1{
+    background-image: url(https://github.com/CodeYourFuture/karma-clone/blob/99d5c3fe44175a5563c36cd7806add33f0c3c670/img/first-background.jpg?raw=true);
```
Comment (ChrisOwen101) [inline css/style.css]: You can use a relative link here

```
	    background-image: url(../img/first-background.jpg);
```

**Comment by ChrisOwen101** [comment]
Not a bad first attempt! HTML/CSS can take a while to master but you're showing a lot of good work in this attempt. 

I'd suggest spending some more time on this to get the website looking more like the original.

My big suggestion is to break the website down into as small parts as you can and try to do them one by one. It's much easier if you can break down problems into smaller parts!

File: css/style.css (hunk 1 above)
Comment (banirezaie) [reply css/style.css]: I will do it for my next works.
thank you for the time you spent on my web.

**Pull request closed without merging**